wiki/geyser/understanding-the-config.md

```diff
@@ -37,6 +39,8 @@ Options for the remote (Java) server.
 
 **Please keep in mind, what you specify in the Geyser `auth-type` option MUST be the same as what the remote server has (with the exception of Geyser being in online mode and remote being in offline mode). You simply cannot join an online mode server without a genuine account. If you want to allow Minecraft: Bedrock Edition accounts to join without a Minecraft: Java Edition account, see the [Floodgate](/wiki/floodgate/) wiki page.**
 
+**`allow-password-authentication`**: Allow for password-based authentication methods through Geyser. Only useful in online mode. If this is false, users must authenticate to Microsoft using a code provided by Geyser on their desktop.
```

Comment:
this option does not exist anymore...

Reply:
It was removed in https://github.com/GeyserMC/Geyser/commit/7983448ce637656db1fca95eb65344a8c6d90de3, but not in the default config. oops

Reply:
> [GeyserMC/Geyser@7983448](https://github.com/GeyserMC/Geyser/commit/7983448ce637656db1fca95eb65344a8c6d90de3)

em,maybe remove it in config

> It was removed in [GeyserMC/Geyser@7983448](https://github.com/GeyserMC/Geyser/commit/7983448ce637656db1fca95eb65344a8c6d90de3), but not in the default config. oops

em, i think remove it in config.yml first,then remove it here

Reply:
The option does nothing, so it can be re-removed on the wiki. I'll PR a removal of the option to Geyser later

Reply:
> The option does nothing, so it can be re-removed on the wiki. I'll PR a removal of the option to Geyser later

ok, maybe i will PR again to remove it, but now i must go to bed, it's 0:24 now :(